apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -54,6 +54,18 @@ export class BookingsRepository_2024_08_13 {
       where: {
         uid: bookingUid,
       },
+      include: {
```

Comment:
Instead of having nested includes check `apps/api/v2/src/modules/credentials/credentials.repository.ts` and see if we can fetch user credentials using that repository instead, and if there is no function then you can add it.

We should not add the nested include because now every place that calls `getByUId` would be slowed down by fetching data it does not need, and here you really are looking to fetch credentials not booking per se so you need to use the credentials repository.